### DISCUSSION TITLE
DBT Tab Not Displayed on Web Interface After Successful Metadata Ingestion

### DISCUSSION THREAD

**Discussion by mazenorabi** [open-metadata/OpenMetadata#16682] [General]
Despite successfully ingesting DBT metadata externally, the DBT tab does not appear on the web interface.
Any reason for that?
- dbt version: 1.8.1
- OpenMetaData version: 1.4.2 
- Running on Linux

**Comment by Prajwal214** [comment]
@mazenorabi You can refer to the below doc
https://docs.open-metadata.org/v1.4.x/connectors/ingestion/workflows/dbt/dbt-troubleshooting#1.-dbt-tab-not-displaying-in-the-ui

**Reply by mazenorabi** [reply]
Thanks @Prajwal214 I checked that and there is no Errors in the log file at all !

**Reply by Prajwal214** [reply]
@mazenorabi Enable debug and run the ingestion again.

**Reply by mazenorabi** [reply]
@Prajwal214 Looks like many Errors as below:
```
 DEBUG    {metadata.Utils:execution_time_tracker:169} - Source.GET executed in 0.03s
 DEBUG    {metadata.Ingestion:metadata:158} - Traceback (most recent call last):
    File "/usr/local/lib/python3.10/site-packages/metadata/ingestion/source/database/dbt/metadata.py", line 150, in get_dbt_owner
   owner = self.metadata.get_reference_by_name(name=dbt_owner)
   TypeError: unhashable type: 'list'
```

**Reply by Prajwal214** [reply]
@mazenorabi Verify manifest.json or catalog.json files
